.babelrc

```diff
@@ -1,3 +1,9 @@
 {
-  "stage": 1
-}
\ No newline at end of file
+  "plugins": [
+    ["transform-runtime", { "polyfill": true, "regenerator": true }]
+  ],
+  "presets": [
+    "es2015",
+    "stage-0"
+  ]
```

Comment:
You can specify multiple presets, but it's best that we stick with just `babel-preset-taskcluster` wherever possible.  babel-compile supports plugins, by including them in a preset.  For a multi-repo team, it's really nice that all the repos use the same babel settings, it makes sure that we can all jump into any repo and know what to expect to be available in the JS.  If there's something beneficial to add to `babel-preset-taskcluster`, I'm pretty sure it'd be easy to include.